packages/bpk-component-carousel/src/BpkCarousel.scss

```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-carousel {
+  display: flex;
+  justify-content: space-between;
+
+  &__wrapper {
+    overflow: hidden;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    flex: 1 0;
+    transition: margin-left $bpk-duration-base;
+  }
+
+  &__item {
+    margin-right: 1.25rem;
```

Comment:
Can we make this use Backpack tokens?

Reply:
We don't have tokens that would make this value hence the hardcoded here